contracts/starknet/interfaces/i_authenticator.cairo

```diff
@@ -0,0 +1,7 @@
+%lang starknet
+
+@contract_interface
+namespace i_authenticator:
```

Comment:
inb4 @bonustrack fires you for writing this 👺👺👺

Reply:
i just love snake case so much, cant help it...